src/lib/Libifl/pbsD_deljoblist.c

```diff
@@ -534,46 +537,41 @@ PBSD_deljoblist(int c, int function, char **jobids, int numjids, char *extend)
 	for (svr_itr = svr_jobid_list_hd; svr_itr != NULL; svr_itr = svr_itr->next) {
 		if (pbs_client_thread_lock_connection(svr_itr->svr_fd) != 0)
 			goto err;
-		for (i = 0; i < svr_itr->total_jobs; i = i + numjids) {
```

Comment:
I'm removing batching and refactoring more of deljoblist as part of my IFL refactoring PR. If you end up reverting the mail suppression related changes, then please also revert the batching changes so that I don't have to resolve a bunch of conflicts with my PR.